travispy/entities/hook.py

```diff
@@ -30,3 +31,29 @@ class Hook(Entity):
         'private',
         'admin',
     ]
+
+    def _set_hook(self, flag):
+        '''
+
+        '''
+        url = self._session.uri + '/hooks/{}'.format(self.id)
+        data = {
+            "hook": {
+                "active": flag
+            }
+        }
+
+        response = self._session.put(url, json = data)
```

Comment:
I started with `data=json.dumps(data)` but I got a `500` error.  Don't know why.  So I ended up using the `json` parameter, documented at [Quickstart — Requests 2.9.1 documentation](http://docs.python-requests.org/en/master/user/quickstart/):

> Instead of encoding the dict yourself, you can also pass it directly using the `json` parameter (added in version 2.4.2) and it will be encoded automatically